src/main/java/fizzbuzz/Main.java

```diff
@@ -0,0 +1,27 @@
+package fizzbuzz;
+
+import java.util.List;
+
+public class Main {
+    
+    public static void testFizzBuzz(String[] args) {
+        if (args.length != 2) {
+            System.err.println("wrong arguments, try: <number> <number>");
+            return;
```

Comment:
This is not really nice because the reader has to think a lot to figure out when and where this method exists. Moreover you have a void return type, therefore do not call `return;`. Refactor this part :-(